etc/picongpu/pizdaint-cscs/picongpu.profile.example

```diff
@@ -70,8 +81,8 @@ export TBG_TPLFILE="etc/picongpu/pizdaint-cscs/normal.tpl"
 # helper tools ################################################################
 
 # allocate an interactive shell for one hour
-#   getInteractive 2  # allocates to interactive nodes (default: 1)
-getInteractive() {
+#   getNode 2  # allocates to interactive nodes (default: 1)
+getNode() {
```

Comment:
@psychocoderHPC still works for me.

as usual:
```bash
getNode
# wait

# srun commandOnNode
srun hostname
```

Reply:
yes it is running I not realized that I am using an old profile.